ruma-serde/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ Breaking changes:
 
 * Remove the `empty` module from the public API
 
+Improvements:
+
+* Add serialization decorator `none_as_empty_string` to serialize `None`s as empty strings
+* Add unit tests for `empty_string_as_none` and `none_as_empty_string`
```

Comment:
Internal changes that don't affect users of the crate shouldn't generally be mentioned in the changelog.

```suggestion
```